index.html

```diff
@@ -13,35 +13,196 @@
+                    </ul>
+                    <button type="button" class="orange-button">See project</button>
+                 
+                </div>
+                
+            </li>
+            <li class="grid-item white project"> 
+                <div class="content">
+                    <h2 class="title">Data Dashboard Healthcare</h2>
+                    <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <ul class="btn">
+                        <li> <button type="button" class="grey-button">HTML</button></li>
+                        <li> <button type="button" class="grey-button">Bootstrap</button></li>
+                         <li><button type="button" class="grey-button">Ruby</button></li>
+                        </ul>
+                         <button type="button" class="orange-button">See project</button>
+                    
+                    </div>
+                    
+                </li>
+                <li class="grid-item white project"> 
+                    <div class="content">
+                        <h2 class="title">Data Dashboard Healthcare</h2>
+                        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                        <ul class="btn">
+                            <li> <button type="button" class="grey-button">HTML</button></li>
+                            <li> <button type="button" class="grey-button">Bootstrap</button></li>
+                             <li><button type="button" class="grey-button">Ruby</button></li>
+                            </ul>
+                             <button type="button" class="orange-button">See project</button>
+                         
+                        </div>
+                        
+                    </li>
+                    <li class="grid-item white project"> 
+                        <div class="content">
+                            <h2 class="title">Data Dashboard Healthcare</h2>
+                            <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                            <ul class="btn">
+                                <li> <button type="button" class="grey-button">HTML</button></li>
+                                <li> <button type="button" class="grey-button">Bootstrap</button></li>
+                                 <li><button type="button" class="grey-button">Ruby</button></li>
+                                </ul>
+                                 <button type="button" class="orange-button">See project</button>
+                             
+                            </div>
+                            
+                        </li>
+
+    </ul>
+
```

Comment:
- [ ] This section seems not to be at the center yet, as previously suggested by a reviewer. You can check his comments [here](https://github.com/sylvester-wamaya/portfolio-website/pull/2#discussion_r1148399900) and screenshots to have a glimpse of what I'm seeing from my end here. Kindly remove all arbitrary values and style accordingly :wink: